### PR TITLE
Add GDPR missing links

### LINKED ISSUE #163: Update links for about Schibsted and on the required fields screen
Links: https://docs.google.com/spreadsheets/d/10kt3MzwG7Gx2vJY3nbVlwYLP7zqXFdaCw425VXY_ZxQ/edit#gid=0

Identification Screen:
What is Schibsted account: P010

Required fields:
adjust your privacy settings at anytime: P007
learn more about Schibsted, data and you: P012

### PR DESCRIPTION
**Pull request by Astach**
Fixes #163

### CODE REVIEW

File: ui/src/main/java/com/schibsted/account/ui/UiUtil.kt [hunk 1]
```diff
@@ -46,4 +53,43 @@ object UiUtil {
...
+    /**
+     * take a text then colorize and underline words in order to get a text looking like a link to click on
+     *
+     * @param fullText the text where we have to find the text to colorize
+     * @param color the color we want to apply
+     * @param textToCustomize the text to colorize
+     * @return [Spannable] the colorized text
+     */
+    @JvmStatic
+    fun getTextAsLink(fullText: String, @ColorInt color: Int, vararg textToCustomize: String): SpannableString {
```
Comment (Nexcius) [inline ui/src/main/java/com/schibsted/account/ui/UiUtil.kt]: Should these be on function?

File: ui/src/main/java/com/schibsted/account/ui/login/screen/information/RequiredFieldsFragment.kt [hunk 2]
```diff
@@ -59,6 +77,35 @@ class RequiredFieldsFragment : FlowFragment<RequiredFieldsContract.Presenter>(),
         })
     }
 
+    private fun getGdprText(): SpannableString {
```
Comment (Nexcius) [inline ui/src/main/java/com/schibsted/account/ui/login/screen/information/RequiredFieldsFragment.kt]: Consider renaming this function

File: ui/src/main/java/com/schibsted/account/ui/login/screen/information/RequiredFieldsFragment.kt [hunk 3]
```diff
@@ -59,6 +77,35 @@ class RequiredFieldsFragment : FlowFragment<RequiredFieldsContract.Presenter>(),
...
+        UiUtil.makeTextClickable(span, privacyText, getLinkAction(privacyLink))
+        UiUtil.makeTextClickable(span, dataUsageText, getLinkAction(dataUsageLink))
+
+        return span
+    }
+
+    private fun getLinkAction(link: String): ClickableSpan {
+        return object : ClickableSpan() {
+            override fun onClick(widget: View) {
+                Logger.debug("lol", "CLICKKKKED")
```
Comment (Nexcius) [inline ui/src/main/java/com/schibsted/account/ui/login/screen/information/RequiredFieldsFragment.kt]: Debug statement

File: ui/src/main/java/com/schibsted/account/ui/login/screen/information/RequiredFieldsFragment.kt [hunk 4]
```diff
@@ -100,11 +147,12 @@ class RequiredFieldsFragment : FlowFragment<RequiredFieldsContract.Presenter>(),
     }
 
     companion object {
-
+        private const val KEY_UI_CONF = "UI_CONF"
```
Comment (Nexcius) [inline ui/src/main/java/com/schibsted/account/ui/login/screen/information/RequiredFieldsFragment.kt]: This should be removed after the config changes are merged

File: ui/src/main/java/com/schibsted/account/ui/login/screen/information/RequiredFieldsFragment.kt (hunk 3 above)
Comment (Astach) [reply ui/src/main/java/com/schibsted/account/ui/login/screen/information/RequiredFieldsFragment.kt]: :laughing:

**Pull request merged**